sklearn/preprocessing/label.py

```diff
@@ -169,7 +169,9 @@ def _encode_check_unknown(values, uniques, return_mask=False):
 
 
 class LabelEncoder(BaseEstimator, TransformerMixin):
-    """Encode labels with value between 0 and n_classes-1.
+    """Encode target labels with value between 0 and n_classes-1. This
```

Comment:
skip line + single backticks for linking to glossary

Reply:
done